jaeger-jaxrs2/src/main/java/com/uber/jaeger/filters/jaxrs2/ClientFilter.java

```diff
@@ -43,10 +45,12 @@
     private final Tracer tracer;
     private final TraceContext traceContext;
     private final Logger logger = LoggerFactory.getLogger(ClientFilter.class);
+    private final Metrics metrics;
 
-    public ClientFilter(Tracer tracer, TraceContext traceContext) {
+    public ClientFilter(Tracer tracer, TraceContext traceContext, StatsFactory statsFactory) {
```

Comment:
actually, does this class need to be public? If it's always created via TraceUtils, it doesn't, which resolves the issue of constructor backwards compatibility